pkg/api/vm/schema.go

```diff
@@ -3,14 +3,14 @@ package vm
 import (
 	"net/http"
 
+	"github.com/rancher/harvester-server/pkg/config"
+	"github.com/rancher/harvester-server/pkg/generated/clientset/versioned/scheme"
+
```

Comment:
Keep the original line break, also check other places

Reply:
this is done in purpose and now all packages are imported and categorized in `build-in, default repo, and the 3rd-party repos`